stable/jenkins/values.yaml

```diff
@@ -97,10 +97,14 @@ master:
   # Enable Kubernetes Liveness and Readiness Probes
   # ~ 2 minutes to allow Jenkins to restart when upgrading plugins. Set ReadinessTimeout to be shorter than LivenessTimeout.
   healthProbes: true
-  healthProbesLivenessTimeout: 90
-  healthProbesReadinessTimeout: 60
-  healthProbeReadinessPeriodSeconds: 10
-  healthProbeLivenessFailureThreshold: 12
+  healthProbesLivenessTimeout: 20
+  healthProbesReadinessTimeout: 20
```

Comment:
Why are you changing the default here? I think that `healthProbesLivenessTimeout` and `healthProbesReadinessTimeout` are by intention higher as there is the option to download plugins via config and that might take some time.

Reply:
Its still in line, the `healthProbesLivenessTimeout` was previously mapping to the initial delay https://github.com/helm/charts/pull/13875/files#diff-3186f811e401dee5654955efc1ba4d42L214

`healthProbesLivenessTimeout` now correctly maps to timeout and an `healthProbeLivenessInitialDelay` attribute was added which [maps to initial delay](https://github.com/helm/charts/pull/13875/files#diff-3186f811e401dee5654955efc1ba4d42R222) and is [set to 120s](https://github.com/helm/charts/pull/13875/files#diff-bd43a39639bf663f4e99c0d186fad542R106) (2m as suggested by the comment) and same for readiness check. Same with initial delay for readiness check which remains as 60s